newsrevenuehub/rev-engine
# [NRH-319] Donations sort order

#### What's this PR do?
The Org admin donations list donations weren't being sorted by "Date" by default, which was annoying. This fixes that.

#### How should this be manually tested?
Go to the org admin donations list and observe that the donations start sorted by -created or "Date" descending

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
